src/core/material-list-api/mutator/fetcher.ts

```diff
@@ -61,6 +65,6 @@ export const fetcher = async <ResponseType>({
 
 export default fetcher;
 
-export type ErrorType<ErrorData> = ErrorData;
+export type ErrorType<ErrorData> = ErrorData & { status: number };
```

Comment:
Please check if we actually need to add anything here to our ErrorType or this change should be removed.